scene/main/http_request.cpp

```diff
@@ -207,6 +207,16 @@ void HTTPRequest::cancel_request() {
 	requesting = false;
 }
 
+Error HTTPRequest::_get_redirect_headers(Vector<String> *r_headers) {
+	for (const String &E : headers) {
+		// We strip content headers when changing a redirect to GET.
+		if (!E.begins_with("Content-Type: ") && !E.begins_with("Content-Length: ") && !E.begins_with("Content-Location: ") && !E.begins_with("Content-Encoding: ") && !E.begins_with("Transfer-Encoding: ") && !E.begins_with("Connection: ") && !E.begins_with("Authorization: ")) {
```

Comment:
This conditional is a bit unwieldy imo. I would suggest pulling it out into a separate function like `_is_content_header` and add some line breaks for readability.